src/chiltepin/jedi/leadtime.py

```diff
@@ -0,0 +1,49 @@
+def leadtime_to_seconds(fcst):
```

Comment:
if we renamed this from `fcst_to_seconds` to `leadtime_to_seconds`. Should we also take advantage of this change and fix the parameter being passed in from `fcst` to `leadtime`?

Reply:
That's a great idea!  It does look confusing now.